### ISSUE TITLE
Live demo link is broken

### ISSUE THREAD

**Issue by RussianCow** [graphql/swapi-graphql#143]
Reposting from graphql/graphiql#834. [The link](https://graphql.org/swapi-graphql) gives a Page Not Found error:

![image](https://user-images.githubusercontent.com/1589155/58657140-82c1e900-82d2-11e9-9a14-a3d1ff11ea88.png)

**Comment by acao** [comment]
thanks @RussianCow . I'll try to track it down

**Comment by acao** [comment]
@RussianCow I was able to figure out how to deploy, and now its back. thanks for catching this. I don't have access to settings so it was quite an unknown for me

**Comment by RussianCow** [comment]
Great, thanks for spinning that up so quickly!

**Comment by acao** [comment]
@RussianCow some folks from the GraphQL Foundation are looking into the https://graphql.org/swapi-graphql/ URL issue itself, I'll keep you posted and close this once it's resolved

**Comment by acao** [comment]
@RussianCow can you confirm that https://graphql.org/swapi-graphql/ works for you now?

**Comment by RussianCow** [comment]
No, I still get the same "Page Not Found" error.

**Comment by acao** [comment]
@RussianCow I think it might be your DNS cache as well, because some folks who didnt originally see this error . Have you tried visiting on, say, a smart phone or another device before, that's not connected to the same router?

**Comment by RussianCow** [comment]
I just tried on my phone with wifi off and got the same error. :/

**Comment by benjie** [comment]
I am also getting the error on my phone which I believe has not visited that page before. If it’s an ISP DNS cache issue I may have to wait 48 hours for it to resolve.

**Comment by benjie** [comment]
Nope, connecting over 4G rather than wifi has same issue.

**Comment by orta** [comment]
I'm also getting the same error - looks like maybe a netlify 404 page?

**Comment by acao** [comment]
I think Facebook is using Netlifly for managing DNS, and theres just a missing or broken DNS entry for swapi-graphql since `dig` doesn't show any A records for it

**Comment by acao** [comment]
The FB folks were saying it looked fine at first when I reported this, but they confirmed it isn't working for them on external networks, so I think its the opposite @benjie , where facebook's ISP DNS cache is giving them a false positive and it's definately down

**Comment by idanlo** [comment]
Any updates?

**Comment by acao** [comment]
We are still looking into it, very actively. In the meanwhile the github.io link above is working!

**Comment by idanlo** [comment]
@acao there is a CORS and `err: BLOCKED BY CLIENT` issue trying to access the github.io link for some reason

**Comment by acao** [comment]
@idanlo this one? https://graphql.github.io/swapi-graphql/

**Comment by acao** [comment]
@idanlo I have very limited access to anything, I don't have access to DNS or settings for any of the github repositories. Only write access to graphiql :( So its been very difficult trying to solve this issue.

**Comment by idanlo** [comment]
@acao yes

**Comment by acao** [comment]
given the nature of the error, it must be that your browser itself is blocking it. potentially an issue with browser plugins? do you have any more verbose console output? this might be a different bug entirely.

**Comment by idanlo** [comment]
@acao after cancelling my adblocker the `err: BLOCKED BY CLIENT` changed to a CORS error

**Comment by acao** [comment]
huh! i have UBlockOrigin, privacy badger, https everywhere etc in firefox and it seems fine. Are you using Chrome or another browser?

**Comment by idanlo** [comment]
Chrome

**Comment by acao** [comment]
@idanlo alas, i cannot reproduce, nor other maintainers at the moment. Would you care to create a seperate issue for this repository?

**Comment by acao** [comment]
This will be a temporary fix until we can track down the issue at the GraphQL Foundation level:
https://github.com/graphql/graphql.github.io/pull/714

**Comment by acao** [comment]
thanks everyone for helping us to track this down. just another hiccup in the tranfer of ownership, still transitioning access/etc. If I had the access I needed I would have fixed this a while ago

**Comment by acao** [comment]
@idanlo do you still see the issue?

**Comment by idanlo** [comment]
yes.
before disabling adblock: 
```OPTIONS https://graphql.github.io/swapi-graphql/ net::ERR_BLOCKED_BY_CLIENT```
after disabling adblock: 
```Access to fetch at 'https://graphql.github.io/swapi-graphql/' from origin '______' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled```

**Comment by acao** [comment]
@idanlo please create a new issue then, as this is unrelated.  the CORS headers are not set by this project, but by github.io itself. this gh pages site is entirely client side.

**Comment by acao** [comment]
@RussianCow this should be fixed as of ~1hr ago, are you able to confirm?

**Comment by acao** [comment]
the issue was, in the transition of graphql.org to netlify from gh-pages, we had left a few things like this hanging that just transparently resolved before because of github.io paths.  It ended up in the ISP DNS cache for a few organizations, so it was hard to track down at first. thanks to everyone who helped solve this issue!

**Comment by RussianCow** [comment]
Yep, works for me!

**Comment by acao** [comment]
@RussianCow no problem! was a good opportunity to pick up a few loose ends from the transition to netlify. sorry for any issues it might have caused. again, thanks for helping us to track this down everyone. @idanlo don't forget to make that issue so we can try and reproduce your issue
